app/assets/stylesheets/admin/loans/_questionnaires.scss

```diff
@@ -277,3 +277,7 @@ section.questionnaires {
   font-style: italic;
   font-weight: bold;
 }
+
+ul.toc {
```

Comment:
thanks for adding class. took me a few to realize toc was table of contents. does that merit adding a quick comment?

Reply:
`toc` is used for table of contents most other places in the code that refer to the table of contents, but I can add a comment if you think it will help